src/main/java/de/ait/secondlife/controllers/OfferController.java

```diff
@@ -217,4 +213,118 @@ public ResponseEntity<ResponseMessageDto> recover(
+                )
+        );
+    }
+
+    @PatchMapping("/{id}/start-auction")
+    @Operation(
+            summary = "Start auction",
+            description = "Transferring an Offer from <i>Verification</i> status to the <i>Auction started</i>. Available only for Admin role and for Offers in <i>Verification</i> status."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successful operation",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "403", description = "Forbidden",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "404", description = "Offer not found",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+    })
+    public ResponseEntity<ResponseMessageDto> startAuction(@PathVariable Long id) {
+        service.startAuction(id);
+        return ResponseEntity.ok(
+                new ResponseMessageDto(
+                        String.format("Auction for offer with id <%s> was started successfully", id)
+                )
+        );
+    }
+
+    @DeleteMapping("/{id}/cancel")
+    @Operation(
+            summary = "Cancel offer by owner",
+            description = "Disabling an Offer. Available only for Offer owners and for Offers in statuses: <i>Verification, Auction started, Auction finished, Qualification</i>."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successful operation",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "403", description = "Forbidden",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "404", description = "Offer not found",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+    })
+    public ResponseEntity<ResponseMessageDto> cancel(@PathVariable Long id) {
+        service.cancelOffer(id);
+        return ResponseEntity.ok(
+                new ResponseMessageDto(String.format("Offer with id <%s> was canceled successfully", id))
+        );
+    }
+
+    @DeleteMapping("/{id}/block-by-admin")
+    @Operation(
+            summary = "Cancel offer by admin",
+            description = "Disabling an Offer. Available only for Admin role and for Offers in statuses: <i>Draft, Verification, Auction started, Auction finished, Qualification</i>."
```

Comment:
Whats the point to block Offer in Draft/Verification statuses for the Admin?  i think the admin cant even view anywhere offers in draft status, and it hasnt a sense to block in verification status, if admin can reject offer and return to offer_owner to remove violations. Also, "Auction_finished" status will be continuing only for a small amount of time (less than a minute maybe), because When the offer reaches the auction_ended time, the status changes automatically to "auction_ended", and in this status it also should be changed to "Qualifiaction" or to "Complete" statute fastly, according to count of bids.